plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -30,32 +30,66 @@
 
 public class CubridPlanNode extends AbstractExecutionPlanNode
 {
-    private static final String OPTIONS_SEPARATOR = ":";
-    private static final String COST = "cost";
-    private static final String CLASS = "class";
+    private static final String SEPARATOR = ":";
+    private static final String SPACE = " ";
+    private static final String NORMAL = "normal";
+    private static final String SINGLE = "single";
+    private static final String MULTIPLE = "multiple";
+
     private static Map<String, String> classNode = new HashMap<>();
     private static Map<String, String> terms = new HashMap<>();
+    private static int i;
```

Comment:
Is it possible to make this field `i` to be a local variable and pass to the methods that need it through arguments?